tests/e2e/features/redirection/mainPage/redirection.feature

```diff
@@ -306,4 +306,26 @@ Feature: External Redirection on the Main Page
     Then Message "Transaction submitted" should be visible
     When I click by text " Explore ecosystem "
     Then New page has "https://ecosystem.zksync.io/" address
+
+  @id1622
```

Comment:
Please add this steps to Allure https://allure.matterlabs.dev/project/17/test-cases/1622?treeId=30

Reply:
Preconditions added in Allure